### PR TITLE
Data augmentation utils

### PR DESCRIPTION
**Pull request by Jeanselme**
Add two primitives for allowing a multi branch predictions in the data augmentation setting

### CODE REVIEW

**Comment by Jeanselme** [comment]
I would need your help to check if they respect the core requirements

**Comment by d8w** [comment]
- [x] May you add your primitive description to README.md?

File: autonbox/merge_multipredictions.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import typing
+import numpy as np
+import pandas as pd
+
+from d3m import container
+from d3m import utils as d3m_utils
+from d3m.metadata import base as metadata_base, hyperparams
+from d3m.primitive_interfaces.base import CallResult
+from d3m.primitive_interfaces.transformer import TransformerPrimitiveBase
+from d3m.primitives.data_preprocessing.horizontal_concat import DSBOX
```
Comment (d8w) [inline autonbox/merge_multipredictions.py]: To delete

**Review by d8w — CHANGES_REQUESTED** [review]
delete `from d3m.primitives.data_preprocessing.horizontal_concat import DSBOX`

File: autonbox/clean_augment.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    """
+        Remove rows which haven't been augmented following the datamart augmentation 
+        (any rows with more than percentage_missing columns which are resulting from augmentation)
+        
+        NB: This primitive results might reduce the number of row
+    """
+
+    metadata = metadata_base.PrimitiveMetadata({
+        'id': 'fe0f1ac8-1d39-463a-b344-7bd498a31b92',
+        'version': '0.1',
```
Comment (d8w) [inline autonbox/clean_augment.py]: Use three part naming conversion for version: i.e. [Major].[Minor].[Patch]
`'version':'0.1.0'`

File: autonbox/merge_multipredictions.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+class MergePartialPredictionsPrimitive(TransformerPrimitiveBase[Inputs, Outputs, hyperparams.Hyperparams]):
+    """
+        Merge predictions of multiple models
+        Useful if model do not produce predictions for each points and that it is necessary
+        to merge those predictions (the first non nan will be returned)
+    """
+
+    metadata = metadata_base.PrimitiveMetadata({
+        'id': '1cc95f70-0716-11ea-9762-3dd2bb86dde8',
+        'version': '0.1',
```
Comment (d8w) [inline autonbox/merge_multipredictions.py]: Use three part naming conversion for version: i.e. [Major].[Minor].[Patch]
`'version':'0.1.0'`

**Review by d8w — CHANGES_REQUESTED** [review]

**Review by d8w — APPROVED** [review]

**Pull request merged**